tasks/release.rake

```diff
@@ -1,6 +1,42 @@
 require "chandler/tasks"
 
+root    = File.expand_path("..", __dir__)
+version = File.read("#{root}/VERSION").strip
+
+# See https://github.com/rails/rails/blob/master/tasks/release.rb#L26
+npm_version = version.gsub(/\./).with_index { |s, i| i >= 2 ? "-" : s }
+
+namespace :activeadmin do
+  desc 'Update and synchronize versions between gem and npm package'
+  task :update_versions do
+    version_file    = File.join(root, 'lib', 'active_admin', 'version.rb')
+    version_content = File.read(version_file)
+
+    version_content.gsub!(/^(\s*)VERSION(\s*)= .*?$/, "\\1VERSION = '#{version}'")
+    raise "Could not insert VERSION in #{version_file}" unless $1
+
+    File.open(version_file, "w") { |f| f.write version_content }
+
+    require "json"
+    if JSON.parse(File.read("#{root}/package.json"))["version"] != npm_version
+      if sh "which npm"
+        sh "npm version #{npm_version} --no-git-tag-version"
+      else
+        raise "You must have npm installed to release ActiveAdmin."
+      end
+    end
+  end
+
+  desc 'Publish npm package'
+  task :npm_push do
+    npm_tag = /[a-z]/.match?(version) ? "pre" : "latest"
```

Comment:
As a followup to the other comment, here we should use `npm`'s criteria for prereleases (which strictly conforms to [semantic versioning](https://semver.org/#spec-item-9). So instead check whether the version string includes a hyphen or not.

Reply:
To clarify this, the current check is perfectly fine, but it seems more clear to me, since we are pushing to `npm`, to use the `npm_version` and the `npm` criteria. But it's just another nitpick, doesn't fix anything since the current check is correct.